internal/file/exclusion_test.go

```diff
@@ -79,6 +79,7 @@ func TestDefaultExclusionsFingerprint(t *testing.T) {
 		filepath.Join("**", ".git", "**"),
 		"**/*.egg-info/**",
 		"**/*venv/**",
+		"**/venv3/**",
```

Comment:
I think Emil (or someone) added \*venv to match anything ending with venv (pyenv etc). My thinking here was that venv3 is more specific. An alternative is just \*venv\* but that seems too inclusive. Basically I thought it safest to just add the one directory type, what do you think? (I can of course change it to \*venv3 and it will almost certainly have the same functionality)

Reply:
That makes sense, just wanted to check in case it was a typo